core/controller/src/main/scala/whisk/core/entitlement/Entitlement.scala

```diff
@@ -303,15 +303,17 @@ protected[core] abstract class EntitlementProvider(config: WhiskConfig, loadBala
   private def checkConcurrentUserThrottle(user: Identity, right: Privilege, resources: Set[Resource])(
     implicit transid: TransactionId): Future[Unit] = {
     if (right == ACTIVATE && resources.exists(_.collection.path == Collection.ACTIONS)) {
-      checkThrottleOverload(concurrentInvokeThrottler.check(user))
+      checkThrottleOverload(concurrentInvokeThrottler.check(user), user)
```

Comment:
You don't need to pass the user through just for logging. Just use the `andThen` combinator:

```scala
checkThrottleOverload(concurrentInvokeThrottler.check(user)).andThen {
  case Failure(_) => logging.debug(this, s"user '${user.subject}' has exceeded its throttle limit")
}
```

Reply:
What you propose is a valid option, however I'd like to leave it as it is to avoid duplicating the log entries. Since we call `checkThrottleOverload` at three places I think that would be better to keep the log entry inside of the method. You could argue that we can create three different log messages for each type of user throttle, but since the message on the debug level, I don't believe that would bring extra value. Passing user as a parameter should not add much overhead, right?

Reply:
i think this is ok, but apropos my message above, having an info level message that user is over quota is helpful.
perhaps rather than user.subject, should the user.uuid instead.